public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -107,39 +98,40 @@ include ../_util-fns
   We put a template expression within the interpolation braces when we wrote `{{1 + 1}}`.
   We’ll see template expressions again in the [property binding](#property-binding) section,
   appearing in quotes to the right of the `=` symbol as in `[property]="expression"`.
-// #enddocregion template-expressions-1
 
-// #docregion template-expressions-2
 :marked
-  We write template expressions in a language that looks like JavaScript.
-  Many JavaScript expressions are legal template expressions, but not all.
-  JavaScript expressions that have or promote side effects are prohibited,
+  We write template expressions in a language that looks like #{_JavaScript}.
+  Many #{_JavaScript} expressions are legal template expressions, but not all.
+  #{_JavaScript} expressions that have or promote side effects are prohibited,
   including:
 
-  * assignments (`=`, `+=`, `-=`)
-  * the `new` operator
-  * chaining expressions with `;` or `,`
+  * assignments (`=`, `+=`, `-=`, ...)
+  * !{__new_op}
+  * chaining expressions with !{__chaining_op}
```

Comment:
Looking into this a bit further ... interestingly, it _is_ possible to chain expressions with side-effects in Dart, using _both_ `;` **and** `,`.

- Using `;` comes from the fact that an expression is a statement -- see [Line 577 in the Dart 1.10 grammar](https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/grammar-AUTOGENERATED-DO-NOT-EDIT.txt#L577).
- Using `,` is possible in the context of `for` loops; see the for-loop production rule on [Line 593](https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/grammar-AUTOGENERATED-DO-NOT-EDIT.txt#L593). The [`expressionList`](https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/grammar-AUTOGENERATED-DO-NOT-EDIT.txt#L199) at the end is a comma-separated list of expressions.

But, comma-separated lists of expressions is not permitted as a statement (in contrast to other C-like languages), so I believe that the prose at line 110 that you refer to above is a good enough short qualification of the situation. Do you agree?